geode-core/src/test/java/org/apache/geode/pdx/JSONFormatterJUnitTest.java

```diff
@@ -30,50 +31,36 @@
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
-import org.apache.geode.cache.AttributesFactory;
 import org.apache.geode.cache.Cache;
 import org.apache.geode.cache.CacheFactory;
 import org.apache.geode.cache.DataPolicy;
 import org.apache.geode.cache.Region;
-import org.apache.geode.cache.RegionAttributes;
-import org.apache.geode.cache.server.CacheServer;
-import org.apache.geode.internal.Assert;
-import org.apache.geode.internal.cache.GemFireCacheImpl;
 import org.apache.geode.pdx.internal.PdxInstanceImpl;
 import org.apache.geode.pdx.internal.PeerTypeRegistration;
-import org.apache.geode.test.junit.categories.FlakyTest;
 import org.apache.geode.test.junit.categories.IntegrationTest;
 import org.apache.geode.test.junit.categories.SerializationTest;
 
 @Category({IntegrationTest.class, SerializationTest.class})
 public class JSONFormatterJUnitTest {
-
-  private GemFireCacheImpl c;
-  private final String PRIMITIVE_KV_STORE_REGION = "primitiveKVStore";
+  public static final String REGION_NAME = "primitiveKVStore";
+  private Cache cache;
+  private Region<Object, Object> region;
 
   @Before
   public void setUp() throws Exception {
-    this.c = (GemFireCacheImpl) new CacheFactory().set(MCAST_PORT, "0").setPdxReadSerialized(true)
-        .create();
-
-    // Create region, primitiveKVStore
-    final AttributesFactory<Object, Object> af1 = new AttributesFactory<Object, Object>();
-    af1.setDataPolicy(DataPolicy.PARTITION);
-    final RegionAttributes<Object, Object> rAttributes = af1.create();
-    c.createRegion(PRIMITIVE_KV_STORE_REGION, rAttributes);
+    this.cache = new CacheFactory().set(MCAST_PORT, "0").setPdxReadSerialized(true).create();
+
+    region = cache.createRegionFactory().setDataPolicy(DataPolicy.PARTITION).create(REGION_NAME);
+
   }
 
   @After
   public void tearDown() {
-    // shutdown and clean up the manager node.
-    this.c.close();
+    this.cache.close();
   }
 
-  private void ValidatePdxInstanceToJsonConversion() {
-
-    Cache c = CacheFactory.getAnyInstance();
-    Region region = c.getRegion("primitiveKVStore");
-
+  @Test
```

Comment:
wow - good catch Galen.  I checked the other JSON unit tests to make sure there weren't similar tests missing "@Test" annotations.

Reply:
Thanks!